eclass/cmake.eclass

```diff
@@ -123,6 +123,11 @@ fi
 # If set, skip detection of CMakeLists.txt unsupported in CMake 4 in case of
 # false positives (e.g. unused outdated bundled libs).
 
+# @ECLASS_VARIABLE: _CMAKE_MINREQVER_UNSUPPORTED
+# @DEFAULT_UNSET
+# @DESCRIPTION:
+# Set to true if an unsupported cmake_minimum_required value was detected.
```

Comment:
Isn't this `@INTERNAL`? Right now it sounds like the user is supposed to set it.